### PR TITLE
[WIP] Add the Wink bindings.

### PR DESCRIPTION
**Pull request by sebmarchand**
This is still a WIP and it really need more documentation, so it's not really ready for review yet (but I'm still open to comments if you want to take a look!).

I'm sending this PR per Kai's suggestion (see https://community.openhab.org/t/offering-to-implement-the-wink-hub-2-bindings/20085/40)

### CODE REVIEW

File: addons/binding/org.openhab.binding.wink/pom.xml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.openhab.binding</groupId>
+    <artifactId>pom</artifactId>
+    <version>2.0.0-SNAPSHOT</version>
```
Comment (martinvw) [inline addons/binding/org.openhab.binding.wink/pom.xml]: Could you rebase (not merge) from the master and make sure this is 2.1.0-SNAPSHOT

**Comment by martinvw** [comment]
Thanks for sharing it!

**Comment by sebmarchand** [comment]
I've rebased this PR as there was a lot of noise in it. I'm not so familiar with PR on Github but it seems like the best thing to do will be to close this PR and create a new one that tracks https://github.com/sebmarchand/openhab2-addons/tree/pull_request/addons/binding/org.openhab.binding.wink (instead of tracking my master branch)

**Comment by sebmarchand** [comment]
Closing this PR, continuing in https://github.com/openhab/openhab2-addons/pull/2317 , sorry for the noise.

**Pull request closed without merging**